Gems/ScriptCanvas/Code/Editor/View/Windows/MainWindow.cpp

```diff
@@ -1652,7 +1653,7 @@ namespace ScriptCanvasEditor
         bool isValidFileName = false;
 
         AZ::IO::FixedMaxPath projectSourcePath = AZ::Utils::GetProjectPath();
-        projectSourcePath /= "ScriptCanvas//";
+        projectSourcePath /= "Assets/ScriptCanvas";
```

Comment:
Hey ! What happens if the Assets/ScriptCanvas folder does not exist ? Isn't the windows file explorer will still go to the root

Reply:
It verifies if it exists, if not, it creates the folder:

```
            QDir dir(suggestedDirectoryPath.c_str());
            if (!dir.exists())
            {
                auto result = AZ::IO::SystemFile::CreateDir(suggestedDirectoryPath.c_str());
                if (!result)
                {
                    AZ_Error("Script Canvas", false, "Failed to make new folder: %s", suggestedDirectoryPath.c_str());
                    return false;
                }
            }
```

Reply:
Oh right I read the review too fast this morning, cool change